Userland/Applications/Browser/Tab.cpp

```diff
@@ -47,6 +55,15 @@ URL url_from_user_input(const String& input)
     StringBuilder builder;
     builder.append("http://");
     builder.append(input);
+    url = URL(builder.build());
+
+    if (url.is_valid() && (may_url_exist(input)) || s_search_engine_format == "{}")
+        return url;
+
+    // Try using search engine
+    dbgln("Using search engine for {}", input);
+    builder.clear();
+    AK::vformat(builder, s_search_engine_format, AK::VariadicFormatParams { input });
```

Comment:
???
`builder.appendff(...)`?

Reply:
`s_search_engine_format` is `"{}"`, so why not just `builder.append(input)`?

Reply:
Oh I didn't know about this :)